src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.cpp

```diff
@@ -192,6 +192,13 @@ void ov::npuw::LLMInferRequest::infer_prefill(ov::SoPtr<ov::ITensor> input_ids,
     LOG_DEBUG("Calling inference for prefill model...");
     LOG_BLOCK();
 
+    auto& kvcache_desc = m_npuw_llm_compiled_model->m_kvcache_desc;
+    if (input_ids->get_size() > kvcache_desc.max_prompt_size) {
+        OPENVINO_THROW("Input prompt is bigger than configured \"NPUW_LLM_MAX_PROMPT_LEN\": ",
+                        kvcache_desc.max_prompt_size, ".\nPlease either setup bigger "
```

Comment:
I'd recommend "longer" instead of "bigger" as we're talking about length here.

Reply:
Fixed